src/types/BannerTypes.ts

```diff
@@ -20,11 +21,12 @@ export type BannerTestTracking = {
     abTestName: string;
     abTestVariant: string;
     campaignCode: string;
+    componentType: OphanComponentType;
+    products?: OphanProduct[];
 };
 
 export type BannerPageTracking = {
     ophanPageId: string;
-    ophanComponentId: string;
```

Comment:
when this change is merged and the backend begins returning `componentType` instead of  `ophanComponentId`, there may be some clients using a cached version of the component which still expects `ophanComponentId`.
Is it worth doing it in 2 steps? Mark `ophanComponentId` as deprecated, but actually remove it another day?

With this new architecture we want the client/fastly to cache the components heavily, so changes to the model should really be backwards compatible for a time.

Reply:
OK I see what you mean, we don't use a hash per build in the `${ComponentName}.js` path so the React Component used by some clients for a time could be a cached version that still requires `ophanComponentId`. In which case I'll temporarily need to continue passing `ophanComponentId` in the `tracking` part of the response.

Reply:
> a hash per build in the ${ComponentName}.js
good idea!
@nicl

Reply:
It'd make development on this more safe, having to accommodate for cached components involves quite some overhead when preparing changes.